L1Trigger/TrackFindingTracklet/src/TrackletEventProcessor.cc

```diff
@@ -245,22 +235,32 @@ void TrackletEventProcessor::event(SLHCEvent& ev) {
             dtcbase = dtc.substr(0, 4) + dtc.substr(6, dtc.size() - 6);
           }
 
-          string fname = "../data/MemPrints/InputStubs/Link_";
+          const string dirIS = settings_->memPath() + "InputStubs/";
+          string fname = dirIS + "Link_";
           fname += dtcbase;
           if (dtcstubs.find(dtcbase + "A") != dtcstubs.end())
             continue;
           fname += "_A.dat";
+
+          if (not std::filesystem::exists(dirIS)) {
+            system((string("mkdir -p ") + dirIS).c_str());
```

Comment:
check if successful

Reply:
DONE
--  P.S. As this file/directory creation code appears in dozens of places, a future PR (almost ready) will move it to a single common function, to reduce duplicate code.